src/Makefile.qt.include

```diff
@@ -234,6 +237,11 @@ RES_ICONS = \
   qt/res/icons/bitcoin.ico \
   qt/res/icons/bitcoin_testnet.ico \
   qt/res/icons/bitcoin.png \
+  qt/res/icons/button_off.png \
+  qt/res/icons/button_on_blue.png \
```

Comment:
Because it's a matter of taste is precisely why it should use standard UI elements. The standard element should do all the drawing. If the user doesn't like their standard widgets, they can change them.